src/lightning/pytorch/loggers/neptune.py

```diff
@@ -551,7 +551,7 @@ def after_save_checkpoint(self, checkpoint_callback: Checkpoint) -> None:
     def _get_full_model_name(model_path: str, checkpoint_callback: Checkpoint) -> str:
         """Returns model name which is string `model_path` appended to `checkpoint_callback.dirpath`."""
         if hasattr(checkpoint_callback, "dirpath"):
-            expected_model_path = f"{checkpoint_callback.dirpath}{os.path.sep}"
+            expected_model_path = os.path.normpath(checkpoint_callback.dirpath) + os.path.sep
```

Comment:
This doesn't seem to work on Windows (traceback: https://github.com/neptune-ai/examples/actions/runs/6691027428/job/18177473941#step:5:419)


```suggestion
            expected_model_path = os.path.normpath(checkpoint_callback.dirpath).replace("/", os.path.sep) + os.path.sep
```

Reply:
Why does it need to suffix a `/` character anyways? If `os.path.join` is used after, then the separator is added automatically

Reply:
It is possible that `model_path` needs to be normpath-ed too for that check to pass